rules/ur/ur-phonetic.js

```diff
@@ -12,20 +12,20 @@
 		version: '1.0',
 		patterns: [
 			[ '!', '!' ],
-			[ '1', '۱' ],
-			[ '2', '۲' ],
+			[ '1', '1' ],
```

Comment:
You can just remove the mapping for the numeral. [ '1', '1' ] has no effect(it means when you type 1, produce 1, which anyway is the default behavior